astroquery/esa/iso/core.py

```diff
@@ -109,6 +145,48 @@ def download_data(self, tdt, *, retrieval_type=None, filename=None,
 
         return filename
 
+    def get_postcard_link(self, tdt, filename=None, verbose=False):    
```

Comment:
filename is not used, and docstring need a cleanup of irrelevant parameter names